tests/cli/test_main_build.py

```diff
@@ -27,7 +27,7 @@ def _reset_config(search_path=None):
 
 
 @pytest.mark.sanity
-def test_build():
+def test_build(conda_build_test_recipe_envvar: str):
```

Comment:
can we rename this.. `test_build` is not completely..

Reply:
I too dislike the name but I opted for it since the cloned repo is called `conda_build_test_recipe`, I considered `cbtr_path` and `cbtr_envvar` but our [styleguide specifies not to use abbreviations](https://github.com/conda/infrastructure/blob/main/STYLEGUIDE.md#72), is there anything else you had in mind?

Reply:
`test_build_path` itself sounds much better.. and adding a  `docstring` could be added to describe the test itself..

Reply:
ooooh you're talking about the test function name itself, not the `conda_build_test_recipe_envvar` fixture 🤦🏼‍♂️

How about `test_build_empty_sections` since this test is only building that one recipe?

```suggestion
def test_build_empty_sections(conda_build_test_recipe_envvar: str):
```

Reply:
or even `test_main_build_empty_sections` since that's the invoked submodule:

```suggestion
def test_main_build_empty_sections(conda_build_test_recipe_envvar: str):
```

Reply:
I like this better..